Tests/ShopifyCheckoutSheetKitTests/CheckoutBridgeTests.swift

```diff
@@ -70,18 +70,30 @@ class CheckoutBridgeTests: XCTestCase {
 		}
 	}
 
-	func testDecodeSupportsCheckoutCompleteEvent() throws {
+	func testDecodeSupportsCheckoutCompletedEvent() throws {
```

Comment:
We already have some test checking the correct behavior of a failure when decoding an event. It's not necessary to add a new one as it will be the same and the functionality that handles failures is related to CheckoutBridge, not the CheckoutCompleted event.

As every field is optional now there's also no needed of adding more tests for partial payload or empty payload.